apps/api/src/controllers/accounts.controller.ts

```diff
@@ -1,7 +1,7 @@
 import { Body, Controller, Get, Post, HttpCode, HttpStatus, Logger, Param, HttpException } from '@nestjs/common';
 import { ApiBody, ApiOkResponse, ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { AccountResponse } from '../../../../libs/common/src/types/dtos/accounts.response.dto';
```

Comment:
can we get rid of all this relative path import stuff? It's easy to set up in tsconfig and jest config. Should be able do something like:
```
import { AccountResponse } from '#common/types/dtos/accounts.response.dto';
```